lib/features/email/presentation/email_view.dart

```diff
@@ -397,7 +397,7 @@ class EmailView extends GetWidget<SingleEmailController> {
                               contentPadding: 0,
                               useDefaultFont: true,
                               scrollController: scrollController,
-                              enableQuoteToggle: isInsideThreadDetailView,
+                              enableQuoteToggle: true,
```

Comment:
So if thread details are disabled or only email is present, then the quote will still be collapsed?